mne/io/eeglab/eeglab.py

```diff
@@ -675,9 +675,12 @@ def read_events_eeglab(eeg, event_id=None, event_id_func='strip_to_integer',
         logger.info('No events found, returning empty stim channel ...')
         return np.zeros((0, 3))
 
-    if (latencies < 0).any():
+    if (latencies < -1).any():
         raise ValueError('At least one event sample index is negative. Please'
                          ' check if EEG.event.sample values are correct.')
+    if (latencies == -1).any():
+        warn("Some events have a sample index of -1. This may cause issues "
+             "during, e.g., resampling.")
```

Comment:
It's actually often due to resampling in eeglab (and having boundary event at the beginning for example :) ).

Reply:
So should I change anything here?

Reply:
I would say something in this flavor: `""At least one event has a sample index of -1. This usually is a consequence of how eeglab handles event latency after resampling - especially when you had a boudary event at the beginning of the file. Please make sure that ..."` I'm just not sure how detailed the 'make sure part' should be. The user could just check whether the events with negative time sample were originally at the beginning of file and are boundary events or events that are not important.